.travis.yml

```diff
@@ -1,3 +1,5 @@
+sudo: required
```

Comment:
In the later stage we will use docker to fetch QGIS Server images, so sudo is required.